schema-source/RSxxSub.schema.yaml

```diff
@@ -0,0 +1,219 @@
+Schema:
+  Object Type: "Meta"
+  References:
+    - "ASHRAE205"
+  Title: "Fenestration - Glazing Substrate"
+  Description: "Schema for ASHRAE 205 annex RSxxSub: Fenestration - Glazing Substrate" 
+  Version: "2.0.0"
+  Root Data Group: "RSxxSub"
+
+# Data Groups
+RSxxSub:
+  Object Type: "Data Group"
+  Data Elements:
+    metadata:
+      Description: "Metadata data group"
+      Data Type: "{Metadata}"
+      Constraints: "schema=RSxxSub"
+      Required: True
+    description:
+      Description: "Data group describing product and rating information"
+      Data Type: "{Description}"
+    performance:
+      Description: "Data group containing performance information"
+      Data Type: "{Performance}"
+      Required: True
+
+Description:
+  Object Type: "Data Group"
+  Data Elements:
+    product_information:
+      Description: "Data group describing product information"
+      Data Type: "{ProductInformation}"
+
+ProductInformation:
+  Object Type: "Data Group"
+  Data Elements:
+    manufacturer:
+      Description: "Manufacturer name"
+      Data Type: "String"
+    model_number:
+      Description: "Model number"
+      Data Type: "Pattern"
+      Notes: "Pattern shall match all model numbers that can be represented by the representation"
+
+Performance:
+  Object Type: "Data Group"
+  Data Elements:
+    thickness:
+      Description: "The total thickness of the glazing."
+      Data Type: "Numeric"
+      Units: "m"
+      Required: True
+    conductivity:
+      Description: "Overall conductivity of the glazing."
+      Data Type: "Numeric"
+      Units: "W/m-K"
+      Required: True
+    front_side_infrared_emissivity:
+      Description: "Front Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: True
+      Constraints: [">=0.0", "<=1.0"]
+    back_side_infrared_emissivity:
+      Description: "Back Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: False
+      Constraints: [">=0.0", "<=1.0"]
+      Notes: "When this data element is missing, the value is assumed to be the same as the front_side_infrared_emissivity."
+    performance_map_optical:
+      Description: "Data group describing optical performance over a range of conditions. A list is provided that corresponds to the lists of wavelength_integration, is_incidence_hemispherical, and is_emergence_hemispherical."
+      Data Type: "[{PerformanceMapOptical}]"
+      Required: True
+    is_incidence_hemispherical:
+      Description: "Indicates that the incidence is hemispherical"
+      Data Type: "[Boolean]"
+      Notes: "True if light arrives from all directions of the hemisphere on the sample surface. If True incidence_angle_polar and incidence_angle_azimuth should not be populated. If False they are expected to be populated."
+      Required: True
```

Comment:
Suggest removing this since the map can indicate constant properties by only providing data at one angle.

Reply:
@nealkruis can you expand a little on how this would be expressed and explained to the vendor? What single angle should be used?